src/lib/ui/app/InsightCard/Editorial.svelte

```diff
@@ -0,0 +1,5 @@
+<script lang="ts">
+  import Svg from '$ui/core/Svg/index.js'
+</script>
+
+<Svg illus id="editorial" w={118} h={40} />
```

Comment:
Component is missing logic which checks if the user is part of Santiment's editorial